content/doc/pipeline/index.adoc

```diff
@@ -18,7 +18,7 @@ Accordingly, pipeline functionality is:
 * Durable: Pipelines can survive both planned and unplanned restarts of your Jenkins master.
 * Pausable: Pipelines can optionally stop and wait for human input or approval before completing the jobs for which they were built.
 * Versatile: Pipelines support complex real-world CD requirements, including the ability to fork or join, loop, and work in parallel with each other.
-* Efficient: Pipelines can restart from any of several saved checkpoints.
+* Efficient: Pipelines using a proprietary plugin can restart from any of several saved checkpoints.
```

Comment:
This whole section should be removed. Highlighting any proprietary products has no place in the Jenkins community documentation.